dodge_the_creeps_2d/core/src/classes/hud.nim

```diff
@@ -4,6 +4,11 @@ import gdextgen/classes/gdSceneTree
 import gdextgen/classes/gdLabel
 # import gdextgen/classes/gdButton
 import gdextgen/classes/gdTimer
+import gdextgen/classes/gdInput
+
+var
+  was_paused: bool = false
+  is_playing: bool = false
```

Comment:
It is more natural for these variables to exist as member variables of a class.

Reply:
I could do so, though needing `self.` for it makes it a bit uglier IMO.

Reply:
I've removed `was_paused` by handling visibility the same way as I did by inverting pausing.

I think structurally `is_playing` makes more sense at top-level as it isn't defining something _about_ `self.Hud`.